cmake/apple/install_apple.sh.in

```diff
@@ -90,6 +94,37 @@ done
 # Finalize .app
 macdeployqt "$APP" $_executables
 
+# Carla is a standalone plugin.  Remove library, look for it side-by-side LMMS.app
+# This MUST be done AFTER calling macdeployqt
+#
+# For example:
+#    /Applications/LMMS.app
+#       Contents/lib/lmms/libcarlabase.dylib
+#       Contents/lib/lmms/libcarlarack.so
+#       Contents/lib/lmms/libcarlapatchbay.so
+#    /Applications/Carla.app
+#       Conents/MacOS/libcarla_standalone2.dylib
+#
+# Note:
+#    If upstream (@falktx) moves or renames this library
+#    pkg-config won't mind, but this section MUST be updated
+#    to reflect the new library path
+#
+libcarla="Carla.app/Contents/MacOS/libcarla_standalone2.dylib"
```

Comment:
I think all `libcarla_standalone2.dylib` here should be changed to `libcarla_native-plugin.dylib` because we switched to `carla-native-plugin`.

Reply:
I'm still working on the linking logic.  I will submit it when I'm done.

Reply:
Linking done via ee6d502.